frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -553,10 +552,6 @@ const AssociatedLandForm: React.FC<IAssociatedLandParentForm> = (
             initialValues={props.initialValues}
             isAdmin={props.isAdmin}
           />
-          <Prompt
```

Comment:
Unfortunately this has too many consequences due to the way we are using the url bar to drive behaviour.

Reply:
PA-2308